disable testing and pushing to quay.io for CentOS7 image

zmiklank opened the pull request:
<!---

Please review the Contribution Guidelines[1] before submitting the Pull Request.

For more information about the Software Collection Organization, please visit the Welcome pages[2].

[1] https://github.com/sclorg/welcome/blob/master/contribution.md
[2] https://github.com/sclorg/welcome

-->

zmiklank commented:
[test-all]

phracek commented on imagestreams/redis-centos.json:
```diff
@@ -9,59 +9,6 @@
-        "referencePolicy": {
-          "type": "Local"
-        },
-        "name": "latest"
```
The latest part should be there and changed to something that is still supported. like 6-el8.

phracek commented on imagestreams/redis-centos.json:
```diff
@@ -9,59 +9,6 @@
-        },
-        "from": {
-          "kind": "ImageStreamTag",
-          "name": "5-el8"
```
`6-el8` should be marked as 'latests'

phracek reviewed (changes requested):
Only one thing. The latest tag should be still present in redis-centos.json file.

Please update also main README.md file and mark CentOS 7 as deprecated or delete it.

zmiklank commented:
> Only one thing. The latest tag should be still present in redis-centos.json file.

This is now updated. Can you recheck?

> Please update also main README.md file and mark CentOS 7 as deprecated or delete it.

THank for catching this. I have updated readme for redis 7. As for other readmes; this PR is based on the nginx one (https://github.com/sclorg/nginx-container/pull/256). The READMEs should be kept as they are, as we still have the centos7 images available on quay.io. We will just stop doing their updates and rebuilds.

phracek reviewed (changes requested):
Only two nitpicks. CentOS 7 is mentioned in main README.md file still as an available image on quay.io

And CentOS 7 is mentioned as supported in the list of supported versions.
I would also prefer in example on main README.md file to replace CentOS 7 with CentOS Stream 8.

zmiklank commented:
> Only one nitpicks.
> 
> And CentOS 7 is mentioned as supported in the list of supported versions. I would also prefer in example on main README.md file to replace CentOS 7 with CentOS Stream 8.

@pkubatrh, could you please share you opinion on this?

phracek commented:
> 

Like @pkubatrh mentioned in nginx-container. Let's keep it.

phracek commented:
[test-all]
